docs/src/explanations/connectable.md

```diff
@@ -43,7 +43,7 @@ section: "chisel3"
    * E.g. `IO(Decoupled(Bool)).ready` is a member of the parent `IO` component
  * "relative alignment" - whether two members of the same component or Chisel type are aligned/flipped, relative to one another
    * see section [below](#alignment-flipped-vs-aligned) for a detailed definition
- * "structural type check" - Chisel type `A` is structurally equivalent to Chisel type `B` if `A` and `B` have matching bundle field names and types (`Record` vs `Vector` vs `Element`), vector sizes, `Element` types (UInt/SInt/Bool/Clock etc))
+ * "structural type check" - Chisel type `A` is structurally equivalent to Chisel type `B` if `A` and `B` have matching bundle field names and types (`Record` vs `Vector` vs `Element`), probe modifiers (probe vs nonprobe), vector sizes, `Element` types (UInt/SInt/Bool/Clock)
```

Comment:
OK, I've snuck in a bit that says, A and B should have matching probe modifiers.